content/blog/info/Informasjon til nye studenter.md

```diff
@@ -6,86 +6,181 @@ important: true
 author: Martin Tverråen
 ---
 
-# II Survival guide: "Det alle vet, men ingen forteller deg"
+# ii Survival guide: "Det alle vet, men ingen forteller deg"
```

Comment:
Kanskje bare kalle den Informatikk Survival Guide som første headline? Om vi skal bruke den for nye studenter er det sikkert litt forvirrende om vi begynner med forkortelser med en gang.